doc/api/domain.md

```diff
@@ -1,6 +1,13 @@
 # Domain
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: `Promise`s created in VM contexts no longer have a `.domain`
```

Comment:
I think the YAML parser has issues with values that start with a backtick? Can you double-check this works?

Reply:
Fixed!